src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/ElasticsearchClientPass.php

```diff
@@ -40,6 +40,12 @@ public function process(ContainerBuilder $container)
             $clientConfiguration['hosts'] = $hosts;
         }
 
+        if ($options = $container->getParameter('api_platform.elasticsearch.options')) {
+            foreach ($options as $option => $value) {
+                $clientConfiguration[$option] = $value;
+            }
+        }
```

Comment:
I'm not a fan of using an array, what options are needed are there many options possible? IMO it's not specific enough.

Reply:
I'm not a great expert of Elastic client, but what I know is that all the options will be passed to `Builder` which has these properties: https://github.com/elastic/elasticsearch-php/blob/e0df11da101135310f91688f17d71fa4b00a2508/src/Elasticsearch/ClientBuilder.php#L105

At this moment I need only one option `sslVerification`.

Reply:
I've taken a second look and this will be just fine as the [`fromConfig`](https://github.com/elastic/elasticsearch-php/blob/e0df11da101135310f91688f17d71fa4b00a2508/src/Elasticsearch/ClientBuilder.php#L192) method will do the checks.